agent/canvas.py

```
@@ -195,7 +195,7 @@ def get_component_name(self, cid):
     def run(self, **kwargs):
         raise NotImplementedError()
 
-    def get_component(self, cpn_id) -> Union[None, dict[str, Any]]:
+    def get_component(self, cpn_id) -> None | dict[str, Any]:
```

Ruff (0.16.1)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)